staging/src/github.com/labring/lvscare/care/iptables.go

```diff
@@ -200,6 +202,33 @@ func (impl *iptablesImpl) ensureIpset() error {
 	return nil
 }
 
+type iptablesRule struct {
+	position utiliptables.RulePosition
+	table    utiliptables.Table
+	chain    utiliptables.Chain
+	args     []string
+}
+
+func (impl *iptablesImpl) extraChainRules() []iptablesRule {
+	rules := make([]iptablesRule, 0)
+	if exists, _ := impl.iptables.ChainExists(utiliptables.TableFilter, "CILIUM_OUTPUT"); exists {
+		// another work around is set `prependIptablesChains=false` in cilium agent's config
```

Comment:
OR maybe this is a common solution? @cuisongliu

Reply:
kube-proxy has extraIP config, don't have extraChainRules config, It's embarrassing, it's constantly being cleaned and rebuilt.